<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerForm</class>
 <widget class="QWidget" name="TimerForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TimerName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TimerCountDown:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="displayCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Displayable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timerNamelineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timerCountDownlineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timerActionlineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[Hide(&quot;timer&quot;), Jump(&quot;gameover&quot;)]</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerDisplayableLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="plainText">
    <string>add DynamicDisplayable(countdown, sec = time) align (0.5, 0.3)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerPyPlainTextEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="plainText">
    <string>init python:                  
    def countdown(st, at, sec):
        rs = sec - st
        rt = Text('Remain {:.1f} Sec'.format(rs), size = 40)
        return (rt, 0.1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dynamic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Displayable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTimerPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Timer Screen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timerPosionlineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>at Position(xalign=.1, yalign=.1)</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hideTimerPushBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide Timer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="effectComboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>464</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>effect:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="timerPositionChkBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>428</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="timerActionCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TimerAction:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addTimerPushButton">
      <property name="text">
       <string>Add Timer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
